ScheduleAPI/Models/IDateNowRetriever.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace ScheduleAPI.Models
+{
+    public interface IDateNowRetriever
```

Comment:
Hate to be a stickler for naming, but can we rename this all to 'IDateTimeNow' it is more precise to what it actually returns.

Reply:
Sure, we can do something like that. I half expected you to change it to something you preferred if you actually merged this.